indico_toolkit/pipelines/file_processing.py

```diff
@@ -45,6 +45,16 @@ def get_file_paths_from_dir(
             f"Found {len(self.file_paths)} valid files and {len(self.invalid_suffix_paths)} paths with invalid suffixes."
         )
 
+    def move_all_file_paths(
```

Comment:
Would you ever want to copy all filepaths instead of moving them?

Reply:
cool idea, I'll look into this